call-home/src/collector/api_models.rs

```diff
@@ -0,0 +1,61 @@
+use serde::Serialize;
```

Comment:
we should be using the openapi spec rather than writing our own, otherwise this will be extremely brittle.